src/GlobalPayments.Api/Terminals/INGENICO/IngenicoVariables.cs

```diff
@@ -0,0 +1,122 @@
+﻿
+using System;
+
+namespace GlobalPayments.Api.Terminals.INGENICO {
+    internal class INGENICO_REQ_CMD {
+        public const string CANCEL = "CMD=CANCEL";
+        public const string DUPLICATE = "CMD=DUPLIC";
+        public const string REVERSE = "CMD=REVERSE";
+        public const string REVERSE_WITH_ID = "CMD=REV{0}";
+        public const string REPORT = "0100000001100826EXT0100000A010B010CMD={0}";
+        public const string RECEIPT = "0100000001100826EXT0100000A010B010CMD={0}";
+    }
+
+    internal class INGENICO_GLOBALS {
+        public const string BROADCAST = "BROADCAST CODE";
+        public const string CANCEL = "CMD=CANCEL";
+        public const string TID_CODE = "TID CODE";
+        public const string KEEP_ALIVE_RESPONSE = "<?xml version=\"1.0\" encoding=\"ISO-8859-1\"?> <TID CODE=\"{0}\">OK</TID>";
+        public static bool KeepAlive = true;
+        public const int IP_PORT = 18101;
+    }
+
+    internal static class INGENICO_RESP {
+        public readonly static string ACKNOWLEDGE = ((char)ControlCodes.ACK).ToString();
+        public readonly static string ENQUIRY = ((char)ControlCodes.ENQ).ToString();
+        public readonly static string NOTACKNOWLEDGE = ((char)ControlCodes.NAK).ToString();
+        public readonly static string ENDOFTXN = ((char)ControlCodes.EOT).ToString();
+        public readonly static string[] XML = { "<CREDIT_CARD_RECEIPT>", "LF" };
+        public readonly static string INVALID = "\u0005\u0004";
+        public readonly static string ENDXML = "</CREDIT_CARD_RECEIPT>";
+    }
+
+    public enum ReceiptType {
+        TICKET,
+        SPLITR,
+        TAXFREE,
+        REPORT
+    }
+
+    public enum ReportType {
+        EOD,
+        BANKING,
+        XBAL,
+        ZBAL
+    }
+
+    public enum Environment {
+        TEST,
+        PRODUCTION
+    }
+    public enum TransactionStatus {
+        SUCCESS = 0,
+        REFERRAL = 2,
+        CANCELLED_BY_USER = 6,
+        FAILED = 7,
+        RECEIVED = 9
+    }
+    public enum ReverseStatus {
+        REVERSAL_SUCCESS = 0,
+        REVERSAL_FAILED = 7,
+        NOTHING_TO_REVERSE = 9
+    }
+    public enum CancelStatus {
+        CANCEL_DONE = 9,
+        CANCE_FAILED = 7
+    }
+
+    public enum DynamicCurrencyStatus {
+        CONVERSION_APPLIED = 1,
+        REJECTED = 0
+    }
+    public enum TransactionSubTypes {
+        SPLIT_SALE_TXN = 0x53, // 0x53 byte for 'S'
+        DCC_TXN = 0x44, // 0x44 byte for 'D'
+        REFERRAL_RESULT = 0x82 // 0x52 byte for 'R'
+    }
+
+    public enum ReportTypes {
```

Comment:
duplicate enum there is another one in the same file called ReportType and has all the same values.

Reply:
I've reviewed the duplicate enum on another file and I believe that it has the same values but I think the only same values on both enums are EOD which is BatchSummary. Can you please advise what are the equivalent for Banking, Z-Bal and X-Bal?